src/pages/Browse/index.spec.tsx

```diff
@@ -149,16 +151,23 @@ describe(__filename, () => {
           }
           return msg;
         }),
-      },
-    };
+      });
+
     const version = {
       ...fakeVersion,
-      entries: {
-        [file]: fakeVersionEntry,
+      file: {
+        ...fakeVersionFile,
+        entries: {
+          [file]: {
+            ...fakeVersionEntry,
+            filename: file,
+            path: file,
```

Comment:
This should be different than `filename`: `filename` should be `test.js` and `path` should be `src/test.js`. I don't know if it is relevant here, but we should not assume both are equal.

Instead of using `file` here, it should be `path` (which is used for the key in `entries` as well as for the `path` field. I would either leave `filename` unchanged or use a `does-not-matter.js` filename.

Reply:
I was confused about that, thanks. I also renamed that variable to `path` which is what it is.